jobs/build-webapp.groovy

```diff
@@ -460,6 +460,11 @@ def deployToGatewayConfig() {
        exec(["make", "-C", "services/graphql-gateway",
              "deploy-gateway-config",
              "DEPLOY_VERSION=${NEW_VERSION}"]);
+       if (fileExists("services/graphql-gateway-2/Makefile")) {
```

Comment:
Is this `Makefile` check to be super extra cautious?  Or is there some situation i which we don't expect the make file to exist?

Reply:
errr - yeah I see you the details in the description had the information I needed.  Tho I think it's helpful to add the context in the code itself. :)